spec/requests/api/v1/provider_spec.rb

```diff
@@ -256,7 +256,7 @@ def get_provider_codes_from_body(body)
       context "with many providers" do
         before do
           101.times do |i|
-            create(:provider, provider_code: "PROV#{i + 1}", age: (200 - i).minutes.ago)
+            create(:provider, provider_code: "PROV#{i + 1}", age: (200 - i).minutes.ago, skip_associated_data: true)
```

Comment:
I still like the idea of separating out the test concerns:

1. a request test that tests that paging works with a `per_page` set to `10`
2. a unit test that only tests that `per_page` defaults to 100, perhaps using mocking if required

Reply:
✅